playkit/src/main/java/com/kaltura/playkit/plugins/ads/ima/IMAConfig.java

```diff
@@ -22,32 +24,41 @@
     public static final String AD_TAG_TIMES             = "tagsTimes";
     public static final String AD_ATTRIBUTION_UIELEMENT = "adAttribution";
     public static final String AD_COUNTDOWN_UIELEMENT   = "adCountDown";
+    public static final String AD_LOAD_TIMEOUT          = "adLoadTimeOut";
 
 
-    private String language = "en";
+    private String language;
     private String adTagURL;
     private boolean enableBackgroundPlayback = true;
     private boolean autoPlayAdBreaks = false;
     private int videoBitrate;
     private boolean adAttribution;
     private boolean adCountDown;
+    private int  adLoadTimeOut;
     private List<String> videoMimeTypes;
     //private Map<Double,String> tagsTimes; // <AdTime,URL_to_execute>
 
     //View companionView;
 
-    public IMAConfig(String language, boolean enableBackgroundPlayback, boolean autoPlayAdBreaks, int videoBitrate, List<String> videoMimeTypes, String adTagUrl, boolean adAttribution, boolean adCountDown) {
+    public IMAConfig(String language, boolean enableBackgroundPlayback, boolean autoPlayAdBreaks, int videoBitrate, List<String> videoMimeTypes, String adTagUrl, boolean adAttribution, boolean adCountDown, int adLoadTimeOut) {
```

Comment:
Instead of a constructor with many many arguments, use a builder.
All setXXX methods return `this`. Then, when creating this object the app can do:
```
imaConfig = new IMAConfig().setLanguage(...).setBackgroundPlayback(...).setAutoPlayAdBreaks(...);
```
Then you don't need the extra constructor.